test/ci/kokoro/linux/run_integ_tests.sh

```diff
@@ -52,7 +55,7 @@ function init_configs {
   # Create .boto config for gsutil
   # https://cloud.google.com/storage/docs/gsutil/commands/config
   "$CFG_GENERATOR" "$GSUTIL_KEY" "$API" "$BOTO_CONFIG"
-  cat "$BOTO_CONFIG" | grep -v private_key
+  grep -v private_key "$BOTO_CONFIG"
```

Comment:
I don't think I understand what this line is doing.  The .boto config file doesn't have a lint that contains "private_key", does it?  Or were you trying to print the JSON keyfile (with private key removed) here?

Reply:
The grep was from before when I was printing the keyfile while debugging, but didn't want the private key in the Fusion logs. I'm a dummy and just kept going "oh yea of course grep out the private key" even for the boto cfg which shouldn't have that field anyways. :)

Reply:
Ok, fixed these both. :)